layouts/partials/analytics.html

```diff
@@ -22,7 +22,8 @@
                     screen: screen.width + "x" + screen.height,
                     user: "{{ .id }}",
                     utcoffset: "1",
-                    })
+                    }),
+                    { referrerPolicy: "no-referrer-when-downgrade" }
```

Comment:
This should be a configurable field with default.

Reply:
I am not used to modifying Hugo themes, but I will try to do it anyway. 

> with default.

What should the default be? With or without the `referrerPolicy`?

Reply:
Ok. I can work on making it configurable. I have check to decide the default value.